scripts/tests/test_agent_step_receipt_relabeling_harness.py

```
@@ -0,0 +1,306 @@
+            "kind"
+        ] = "proof"
+        for entry in bundle["evidence_manifest"]["entries"]:
+            if entry["corresponding_receipt_field"] == "/model_receipt_commitment":
+                entry["evidence_kind"] = "proof"
+        HARNESS.recompute_manifest_commitments(bundle)
+
+        with self.assertRaisesRegex(HARNESS.AgentReceiptError, "lacks compatible evidence"):
+            HARNESS.verify_bundle(bundle)
+
+    def test_top_level_non_object_rejects_without_raw_exception(self) -> None:
+        with self.assertRaisesRegex(HARNESS.AgentReceiptError, "bundle must be an object"):
```

Ruff (0.15.12)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)